Google.PowerShell.IntegrationTests/Logging/Logging.GcLogSink.Tests.ps1

```diff
@@ -194,14 +199,14 @@ Describe "New-GcLogSink" {
         }
     }
 
-    It "should work with -NoUniqueWriterIdentity" {
+    It "should work with -UniqueWriterIdentity" {
```

Comment:
This cmdlet is in the beta module (not the GA one) so that should not be a problem.